data_model/src/account.rs

```diff
@@ -124,9 +125,19 @@ impl Default for SignatureCheckCondition {
 /// Builder which should be submitted in a transaction to create a new [`Account`]
 #[allow(clippy::multiple_inherent_impl)]
 #[derive(
-    Debug, Display, Clone, PartialEq, Eq, Decode, Encode, Deserialize, Serialize, IntoSchema,
+    Debug,
+    Display,
+    Clone,
+    OrdEqHash,
```

Comment:
I'd add the information, that it's working only with `id` field in the name of this derive. Something like `IdBasedComparision`

Reply:
Not sure about this idea

Reply:
Changed it to `IdOrdEqHash` as the two macros have been merged.